app/models/journaled/bulk_delivery.rb

```diff
@@ -7,7 +7,8 @@ def initialize(records:, app_name:)
   def perform
     return unless Journaled.enabled?
 
-    kinesis_client.put_records request
+    response = kinesis_client.put_records request
+    reenqueue_failed_records!(response)
```

Comment:
Nit: My language-driven eye will not give me respite until I ask of thee:
   Could this be just `requeue`?

Reply:
That's one of the things that's easier to do in the office: linguistic-technical ad-hoc discussions

Reply:
🤣  Ya I think that reads a lot better!

Reply:
it's fewer letters and grammatically correct, love `requeue`